cmds/core/yes/yes_test.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2017-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bufio"
+	"testing"
+
+	"github.com/u-root/u-root/pkg/testutil"
+)
+
+func TestMain(m *testing.M) {
+	testutil.Run(m, main)
+}
+
+func TestYes(t *testing.T) {
```

Comment:
Looks much better! One more improvement you could make. Take a look at:
https://github.com/linuxboot/fiano/blob/57266e68c811cb6cf84f1dc89c85a5bdbb1525f4/pkg/uefi/section_test.go#L26

Notice how we can give names to tests and run them? this allows you to be more explicit about which subtest failed

Reply:
That is a neat idea. I will definitely start to use this more often when I have table driven tests! :)

Reply:
This is a great way to get a better idea of where the test fails. I actually failed my tests and it catches each test that failed and tells me the name of each of the failed test. That is very neat. Will start utilizing this more often! Thanks, Gan!